e2e/content-services/document-list/document-list-actions.e2e.ts

```diff
@@ -188,17 +201,166 @@ describe('Document List Component - Actions', () => {
+            done();
+        });
+
+        beforeEach(async (done) => {
+            loginPage.loginToContentServicesUsingUserModel(contentServicesUser);
+            contentServicesPage.goToDocumentList();
+            contentServicesPage.waitForTableBody();
+            paginationPage.selectItemsPerPage('5');
+            contentServicesPage.checkAcsContainer();
+            contentListPage.waitForTableBody();
+            expect(paginationPage.getCurrentItemsPerPage()).toEqual('5');
+            expect(paginationPage.getPaginationRange()).toEqual('Showing 1-' + 5 + ' of ' + 6);
+            done();
+        });
+
+        afterAll(async (done) => {
+            await alfrescoJsApi.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+            await folders.forEach(function (folder) {
+                uploadActions.deleteFilesOrFolder(alfrescoJsApi, folder.entry.id);
+            });
+            done();
+        });
+
+        it('[C260132] Move action on folder with - Load more', () => {
+
+            contentListPage.rightClickOnRow('A' + folderModel1.name);
+            contentServicesPage.checkContextActionIsVisible('Move');
+            contentServicesPage.pressContextMenuActionNamed('Move');
+            copyMoveDialog.checkDialogIsDisplayed();
+            expect(copyMoveDialog.getDialogHeaderText()).toBe('Move \'' + 'A' + folderModel1.name + '\' to...');
+            copyMoveDialog.checkSearchInputIsDisplayed();
+            expect(copyMoveDialog.getSearchLabel()).toBe('Search');
+            copyMoveDialog.checkSelectedSiteIsDisplayed('My files');
+            copyMoveDialog.checkCancelButtonIsDisplayed();
+            copyMoveDialog.checkMoveCopyButtonIsDisplayed();
+            expect(copyMoveDialog.getMoveCopyButtonText()).toBe('MOVE');
+            expect(copyMoveDialog.numberOfResultsDisplayed()).toBe(5);
+            copyMoveDialog.clickLoadMoreButton();
+            expect(copyMoveDialog.numberOfResultsDisplayed()).toBe(6);
+            copyMoveDialog.checkLoadMoreButtonIsNotDisplayed();
+            copyMoveDialog.selectRow('F' + folderModel6.name);
+            copyMoveDialog.checkRowIsSelected('F' + folderModel6.name);
+            copyMoveDialog.clickCancelButton();
+            copyMoveDialog.checkDialogIsNotDisplayed();
+            contentServicesPage.checkContentIsDisplayed('A' + folderModel1.name);
+
+            contentListPage.rightClickOnRow('A' + folderModel1.name);
+            contentServicesPage.checkContextActionIsVisible('Move');
+            contentServicesPage.pressContextMenuActionNamed('Move');
+            copyMoveDialog.checkDialogIsDisplayed();
+            copyMoveDialog.clickLoadMoreButton();
+            copyMoveDialog.selectRow('F' + folderModel6.name);
+            copyMoveDialog.checkRowIsSelected('F' + folderModel6.name);
+            copyMoveDialog.clickMoveCopyButton();
+            notificationPage.checkNotifyContains('Move successful');
```

Comment:
I am facing error on asserting the notification message - 'Move successful'
 [C260132] Move action on folder with - Load more (16 secs)
        - Failed: Element is not visible by.cssContainingText("simple-snack-bar", "Move successful")
        Wait timed out after 7005ms
@adinapitul let me know, if you have any suggestions?